pkg/ddl/index_cop.go

```diff
@@ -280,8 +281,8 @@ func buildTableScan(ctx context.Context, c *copr.CopContextBase, startTS uint64,
 		SetStartTS(startTS).
 		SetKeyRanges([]kv.KeyRange{{StartKey: start, EndKey: end}}).
 		SetKeepOrder(true).
-		SetFromSessionVars(c.SessionContext.GetDistSQLCtx()).
-		SetFromInfoSchema(c.SessionContext.GetDomainInfoSchema()).
+		SetFromSessionVars(c.DistSQLCtx).
+		SetFromInfoSchema(c.InfoSchema).
```

Comment:
`c.InfoSchema` comes from `mock.Context` and is always a "mock"? Could we just set an empty infoschema with version `0`? @tangenta

Reply:
This field doesn't seem to be used in TiKV. We can remove `SetFromInfoSchema`.